src/components/Playground/PlaygroundActions.js

```diff
@@ -30,6 +30,10 @@ const getPlaygroundsQuery = gql`
       lat
       status
       volunteerCount
+      volunteers {
```

Comment:
This query is meant to get the summary of all playgrounds. You should not include these details as it can cause thousands of volunteers to be included in the query. (And we poll to keep everything up-to-date, so it can cause quite some overhead.) 
See also my comment below which will stop you from requiring this

Reply:
Thanks! It very important remark.